### PR TITLE
Update Resources

### PR DESCRIPTION
**Pull request by mbfrahry**
This PR takes care of some issues I found and adds some improvements

### CODE REVIEW

**Comment by mbfrahry** [comment]
retest this please

**Comment by mbfrahry** [comment]
@wenlock, This should be good to merge when you are ready

File: ov/power_state.go [hunk 1]
```diff
@@ -147,7 +148,7 @@ func (pt *PowerTask) SubmitPowerState(s PowerState) {
 	if s != pt.State {
 		log.Infof("Powering %s server %s for %s.", s, pt.Blade.Name, pt.Blade.SerialNumber)
 		var (
-			body = PowerRequest{PowerState: s.String(), PowerControl: P_MOMPRESS.String()}
+			body = PowerRequest{PowerState: s.String(), PowerControl: P_PRESSANDHOLD.String()}
```
Comment (wenlock) [inline ov/power_state.go]: this should address https://github.com/HewlettPackard/docker-machine-oneview/issues/86 also

**Comment by wenlock** [comment]
im ready to merge this one also, just need it rebased

**Comment by mbfrahry** [comment]
@wenlock looks good to go!

**Comment by wenlock** [comment]
LGTM, thanks!

**Pull request merged**